frontend/scripts/react-components/shared/tooltip/tooltip.component.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './tooltip.scss';
+import { Manager, Reference, Popper } from 'react-popper';
+
+function Tooltip({ reference, referenceClassName, children, type }) {
```

Comment:
is this file necessary?

Reply:
I'm afraid yLabelsProfileInfo is not available from the backend in multiyear charts 😕